core/iwasm/interpreter/wasm_interp_fast.c

```diff
@@ -6435,10 +6435,37 @@ wasm_interp_call_func_bytecode(WASMModuleInstance *module,
                     case SIMD_v128_store16_lane:
                     case SIMD_v128_store32_lane:
                     case SIMD_v128_store64_lane:
+                    {
+                        wasm_set_exception(module, "unsupported SIMD opcode");
+                        break;
+                    }
+#define SIMD_LOAD_ZERO_OP(width)                   \
```

Comment:
It appears there is some code shared with `SIMD_LOAD_LANE_OP`(like after `int32 base...`). Is there a way we can minimize code duplication?

Reply:
yeah that's a great idea actually 

what we can do is call `load_lane (0, width)` to load the 0 lane and then nullify other lanes